src/base58.c

```diff
@@ -44,8 +45,9 @@ int btc_base58_decode(void* bin, size_t* binszp, const char* b58)
     size_t binsz = *binszp;
     const unsigned char* b58u = (const void*)b58;
     unsigned char* binu = bin;
-    size_t outisz = (binsz + 3) / 4;
-    uint32_t outi[outisz];
+    const size_t outisz = (binsz + 3) / 4;
+    const size_t outilen = sizeof(uint32_t) * outisz;
+    uint32_t *outi = (uint32_t *)alloca(outilen);
```

Comment:
Hi Jonas,
alloca() doesn't heap allocates - it allocates data on stack instead, and it appears no btc_alloca() in the code. It replaces VLA (supported only by gcc and C99 and discouraged by clang, for example). Visual Studio doesn't support VLAs, that's why I submitted the PR